client/bin/setup.sh

```diff
@@ -35,7 +35,8 @@ oc secrets add serviceaccount/builder secrets/kojisecret --for=mount
 
 oc adm policy add-role-to-user system:build-strategy-custom -z builder -n osbs
 
-oc secrets new-dockercfg v2-registry-dockercfg --docker-server=172.17.0.1:5000 --docker-username=osbs --docker-password=craycray --docker-email=test@test.com
+oc create secret generic v2-registry-dockercfg --from-file=.dockerconfigjson=/configs/dockerconfigjson --type=kubernetes.io/dockerconfigjson
```

Comment:
Had to disable auth on the local registry, see https://github.com/lcarva/osbs-box/commit/848d3574a7fe75e4707dd8dec629dbf74ecead4b

I don't think this commit is needed anymore.

Reply:
Or maybe leave it here to make it easier for someone to test different scenarios. Either one works for me.

Reply:
You are probably right, I'm now testing if it could be dropped completely.

Reply:
I have just tested and it seems that the osbs still needs this secret, even with changes to the osbs.conf disabling auth.